### PR TITLE
Update package.json

### PR DESCRIPTION
**Pull request by SrinivasanTarget**
@marekcirkos Please review

### CODE REVIEW

File: Inspector/package.json [hunk 1]
```diff
@@ -2,8 +2,8 @@
   "name": "web-driver-inspector",
   "version": "1.0.0",
   "description": "",
```
Comment (lawrencelomax) [inline Inspector/package.json]: This is where you want to have 

`'React based web inspector to inspect current device state'`

File: Inspector/package.json [hunk 2]
```diff
@@ -2,8 +2,8 @@
...
-  "main": "index.js",
-  "author": "",
+  "main": "index.html",
+  "author": "React based web inspector to inspect current device state",
```
Comment (lawrencelomax) [inline Inspector/package.json]: Author should be `Facebook, Inc.`

File: Inspector/package.json [hunk 3]
```diff
@@ -2,8 +2,8 @@
   "name": "web-driver-inspector",
   "version": "1.0.0",
   "description": "",
-  "main": "index.js",
-  "author": "",
+  "main": "index.html",
```
Comment (marekcirkos) [inline Inspector/package.json]: Did you check if clean build inspector works afterwards?

**Comment by facebook-github-bot-6** [comment]
@SrinivasanTarget updated the pull request.

File: Inspector/package.json (hunk 3 above)
Comment (SrinivasanTarget) [inline Inspector/package.json]: It works fine for me

**Comment by marekcirkos** [comment]
@facebook-github-bot shipit

**Comment by SrinivasanTarget** [comment]
@marekcirkos On a different context, why we didn't publish web-driver-inspector in npm?

**Comment by facebook-github-bot-9** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://our.intern.facebook.com/intern/opensource/github/pull_request/889894041136925/int_phab) to review.

**Pull request closed without merging**